js/app/demo.js

```diff
@@ -1,6 +1,6 @@
 // Code goes here
 
-require.config({
```

Comment:
Appeared to be needed, though not sure why exactly. May be a conflict with a diff version of require being used somewhee or something, but this works